src/OpenAnomalyDetection_test.py

```diff
@@ -4,16 +4,17 @@
 
 def test_detect():
     input = dict()
-    input['data_path'] = 'data://TimeSeries/GenerativeForecasting/formatted_data_rossman_10.json'
-    input['model_input_path'] = 'file://tmp/rossman_0.1.0.zip'
-    input['graph_save_path'] = 'file://tmp/graph_file.png'
-    input['max_sigma'] = 2.0
-    input['variable_index'] = 4
+    input['data_path'] = 'data://TimeSeries/GenerativeForecasting/m4_daily.json'
+    # input['model_input_path'] = 'file://tmp/m4_daily_0.1.0.zip'
```

Comment:
Is this commented line needed?

Reply:
you're right, I'm actually going to create an "algorithmia compatible" and another version that's entirely local - that will be a cleaner experience.